components/channel_layout/channel_identifier_router/actions.js

```diff
@@ -265,7 +266,24 @@ function handleError(match, history, defaultChannel) {
     history.push(team ? `/${team}/channels/${defaultChannel}` : '/');
 }
 
+function isDirectChannelIdentifier(identifier) {
+    return identifier.length === LENGTH_OF_USER_ID_PAIR && USER_ID_PAIR_REGEXP.test(identifier);
```

Comment:
Yes but as we are not going to remove that const we can still use it just in case in the future we have another routing for 54 chars channels or something

Reply:
wouldn't that have a separate regex then that should also check for size? or is this a change that is coming soon?

Reply:
I use it to avoid calling the regexp, using the shortcircuit and, if not necessary